src/main/java/vfyjxf/bettercrashes/upload/CrashReportUpload.java

```diff
@@ -0,0 +1,31 @@
+package vfyjxf.bettercrashes.upload;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+import vfyjxf.bettercrashes.BetterCrashes;
+import vfyjxf.bettercrashes.BetterCrashesConfig;
+
+public class CrashReportUpload {
+
+    private static final Map<String, IUploadService> services = new HashMap<>();
+    private static final String DEFAULT_SERVICE = "mclo.gs";
+
+    static {
+        services.put("mclo.gs", new MclogsUploadService());
```

Comment:
I did it deliberately like this. The idea is, in case support for other services is added, they are all listed in the static block like

```java
static {
    services.put("mclo.gs", new MclogsUploadService());
    services.put("pastebin.com", new ...);
    services.put("paste.ee", new ...);
    // ...
}
```

That way all the keys for the services (that are also used in the config option) are listed like that. The `DEFAULT_SERVICE` constant is not meant to save a string but rather point to one of the service keys.

Reply:
Oh I see, that's fine then